kymatio/frontend/torch_frontend.py

```diff
@@ -0,0 +1,25 @@
+import torch.nn as nn
+
+class Scattering(nn.Module):
+   def __init__(self, J, shape, max_order = None):
+       super(Scattering, self).__init__()
+       self.J = J
+       self.shape = shape
+       self.max_order = max_order
+
+   def build(self):
+       self.filters = self.create_and_register_filters()
+       raise NotImplementedError
```

Comment:
I think you might have to instantiate the error. Otherwise you'll get an error while raising the error :D

Reply:
Both work: https://stackoverflow.com/questions/13052991/proper-way-to-raise-exception-in-python